nomad/plan_apply.go

```diff
@@ -737,6 +737,11 @@ func evaluateNodePlan(snap *state.StateSnapshot, plan *structs.Plan, nodeID stri
 			return true, "", nil
 		}
 		return false, "node is disconnected and contains invalid updates", nil
+	} else if node.Status == structs.NodeStatusDown {
+		if isValidForLostNode(plan, node.ID) {
+			return true, "", nil
+		}
+		return false, "node is lost and contains invalid updates", nil
```

Comment:
```suggestion
		if isValidForDownNode(plan, node.ID) {
			return true, "", nil
		}
		return false, "node is down and contains invalid updates", nil
```

Is `lost` a valid status for nodes? I think it only applies to allocs.